extensions/amp-consent/0.1/amp-consent.css

```diff
@@ -96,3 +96,22 @@ amp-consent.i-amphtml-consent-ui-in.i-amphtml-consent-ui-iframe-fullscreen > .i-
   height: 100vh !important;
 }
 
+@keyframes i-amphtml-consent-ui-mask {
+  0% {opacity: 0.0;}
+  100% {opacity: 0.2;}
+}
+
+.i-amphtml-consent-ui-mask {
+  position: fixed !important;
+  top: 0 !important;
+  left: 0 !important;
+  width: 100vw !important;
+  height: 100vh !important;
+  opacity: 0.2;
+  animation: i-amphtml-consent-ui-mask 0.5s ease-in;
+  /* Prevent someone from making this a full-sceen image */
+  background-image: none !important;
+  background-color: #000;
+  z-index: 2147483644;
```

Comment:
Could you update the https://github.com/ampproject/amphtml/blob/master/css/Z_INDEX.md